lib/ae_mdw/node.ex

```diff
@@ -0,0 +1,147 @@
+defmodule AeMdw.Node do
```

Comment:
Wonderful, this will clean some warnings too. 
Wouldn't the calls need a dispatch to the Smart instance?

Reply:
This module is compiled and then loaded before the app starts, then the app overrides everything in it once it starts. This can be [seen here](https://github.com/ks/smart_global/blob/master/lib/smart_global.ex#L68)